packages/compose/src/hooks/use-drop-zone/index.js

```diff
@@ -140,10 +149,28 @@ export default function useDropZone( {
 
 			function onDragOver( /** @type {DragEvent} */ event ) {
 				// Only call onDragOver for the innermost hovered drop zones.
-				if ( ! event.defaultPrevented && onDragOverRef.current ) {
-					onDragOverRef.current( event );
+				if (
+					! event.defaultPrevented &&
+					onDragOverRef.current &&
+					dragTimingRef.current
+				) {
+					const time = Date.now();
+					if ( time - dragTimingRef.current.timestamp > 16 ) {
+						const distance =
+							Math.abs(
+								event.clientY - dragTimingRef.current.dragY
+							) +
+							Math.abs(
+								event.clientX - dragTimingRef.current.dragX
+							);
+						if ( distance < 10 ) {
```

Comment:
I guess the way this stands this is really a speed calculation. The cursor has to travel at slower than 10pixels per ~16ms to trigger an update.